db/compaction/compaction_picker_universal.cc

```diff
@@ -960,7 +960,7 @@ Compaction* UniversalCompactionBuilder::PickDeleteTriggeredCompaction() {
       GetCompressionType(ioptions_, vstorage_, mutable_cf_options_,
                          output_level, 1),
       GetCompressionOptions(mutable_cf_options_, vstorage_, output_level),
-      /* max_subcompactions */ 0, /* grandparents */ {}, /* is manual */ true,
+      /* max_subcompactions */ 0, /* grandparents */ {}, /* is manual */ false,
```

Comment:
This needs to be checked and fixed for level compaction as well, so the same compaction reason is consistently reported for both level and universal.